pytest_mysql/executor.py

```
@@ -125,42 +127,53 @@ def initialise_mysql_db_install(self) -> None:
+            subprocess.check_output(shutdown_command)
         except subprocess.CalledProcessError:
             # Fallback to using root user for shutdown
-            shutdown_command = f"{self.admin_exec} --socket={self.unixsocket} --user=root shutdown"
-            subprocess.check_output(shutdown_command, shell=True)
+            shutdown_command = [
+                self.admin_exec,
+                "--socket={self.unixsocket}",
+                "--user=root",
+                "shutdown",
+            ]
+            subprocess.check_output(shutdown_command)
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)